<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslatorDialog</class>
 <widget class="QDialog" name="TranslatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-30</y>
     <width>941</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../doujzz.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="sourceTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="targetTextEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>331</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="translateButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Traduire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 16pt &quot;Rockwell Extra Bold&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>traduction:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sourceLanguageComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="targetLanguageComboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>choisir la langue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>choisir la langue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
